src/class/msc/msc_device.c

```diff
@@ -74,8 +81,7 @@ typedef struct
   uint8_t add_sense_qualifier;
 }mscd_interface_t;
 
-CFG_TUD_MEM_SECTION CFG_TUSB_MEM_ALIGN tu_static mscd_interface_t _mscd_itf;
-CFG_TUD_MEM_SECTION CFG_TUSB_MEM_ALIGN tu_static uint8_t _mscd_buf[CFG_TUD_MSC_EP_BUFSIZE];
+CFG_TUD_MEM_SECTION CFG_TUD_MEM_ALIGN static mscd_interface_t _mscd_itf;
```

Comment:
Not sure, if we need to apply the whole `CFG_TUD_MEM_SECTION` to the interface struct, as it is literally required only for the buffers.

Reply:
it is requierd for the cbw, csw